### DISCUSSION TITLE
"Join directly" setting

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#549] [Q&A] [answered]
Really nice update! Quick question: I am trying to find where I can toggle the setting to let people directly join our hub. Can’t seem to find it.

**Comment by techsmyth** [comment]
Hi Flori, this is not yet there on the client admin UI, but is pretty much at the top of the backlog: https://github.com/alkem-io/client-web/issues/1872 - next sprint...

**Comment by techsmyth** [comment, marked as answer]
Hi Flori, this is now available on the Hub Admin community tab. Cheers, Neil
